docs/configuration.md

```diff
@@ -112,6 +112,7 @@ Sugoi-api implements spring security with full customization allowed
 | fr.insee.sugoi.api.regexp.role.admin                       |                                                                                                                        |               |         |
 | fr.insee.sugoi.ldap.default.group_manager_source_pattern                | Default pattern to use when searching manager group for application. Application name should be passed via {appliname} |               |         |
 | fr.insee.sugoi.api.regexp.role.password.manager            |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.regexp.role.password.validator          |                                                                                                                        |               |         |
```

Comment:
Je propose : | fr.insee.sugoi.api.regexp.role.password.validator    | Default pattern to use when searching password manager in a realm or userstorage. Userstorage should be passed as\$(realm) and userstorage as\$(userstorage) | no default   |   PASSWORD\_\VALIDATOR\_\$(realm),PASSWORD\_VALIDATOR\_\$(realm)\_$(userstorage)  |